templates/all-services/reports/a11y-1.json

```diff
@@ -0,0 +1,648 @@
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:meaningful-sequence",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The order of the content has meaningful order.",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:sensory-characteristics",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:orientation",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The success criterion was tested in Firefox Responsive Design Mode. Content does not restrict its view and operation to a single display orientation, such as portrait or landscape.",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:identify-input-purpose",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:use-of-color",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The success criterion was tested with WAVE. The WAVE browser extensions allows to evaluate web content for accessibility issues directly within a browser.",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:audio-control",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:contrast-minimum",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The contrast ratio was tested with WCAG Contrast checker a FrieFox add-on",
```

Comment:
Fix typo, "FrieFox" changed to "Firefox".

Reply:
fixed